internal/pkg/stream/ecs.go

```diff
@@ -152,7 +152,8 @@ func (s *ECSDeploymentStreamer) Fetch() (next time.Time, done bool, err error) {
 		if _, ok := s.pastEventIDs[id]; ok {
 			break
 		}
-		if msg := aws.StringValue(event.Message); isFailureServiceEvent(msg) {
+		msg := aws.StringValue(event.Message)
+		if isFailureServiceEvent(msg) {
```

Comment:
why do we make this change 🤔

Reply:
Oh, this was a lingering change when I had rollover events listed separately! Reverting.